Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/SyncKronosToShiftsController.cs

```diff
@@ -89,131 +88,146 @@ public ActionResult SyncKronosToShifts(string isRequestFromLogicApp)
 
             this.telemetryClient.TrackTrace($"SyncKronosToShifts starts at: {DateTime.UtcNow.ToString("O", CultureInfo.InvariantCulture)}", telemetryProps);
 
-            this.taskWrapper.Enqueue(this.ProcessKronosToShiftsShiftsAsync(isRequestFromLogicApp));
+            if (bool.TryParse(isRequestFromLogicApp, out bool isFromLogicApp))
+            {
+                this.taskWrapper.Enqueue(this.ProcessKronosToShiftsShiftsAsync(isRequestFromLogicApp));
 
-            using (StringContent stringContent = new StringContent(string.Empty))
+                using (StringContent stringContent = new StringContent(string.Empty))
```

Comment:
With this using statement, you just left the ActionResult that you returned with a disposed StringContent.

Reply:
@aosolis , Removed the using statement and sent string.Empty() to ActionResult.